docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  mysqldb:
+    image: mysql:8
+    container_name: mysql_hometask_5and6
+    restart: always
+    env_file: .env
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQLDB_PASSWORD}
+      MYSQL_DATABASE: ${MYSQLDB_DATABASE}
+      MYSQL_USER: ${MYSQLDB_USER}
+      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
+    ports:
+      - "${MYSQLDB_LOCAL_PORT}:${MYSQLDB_DOCKER_PORT}"
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-h", "localhost", "-P", "${MYSQLDB_DOCKER_PORT}", "-u", "${MYSQLDB_USER}", "-p${MYSQLDB_PASSWORD}"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+    volumes:
+      - ./db_init.sh:/docker-entrypoint-initdb.d/db_init.sh
+
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      mysqldb:
+        condition: service_healthy
+    restart: on-failure
+    image: andrua
+    container_name: api_hometask_5and6
+    env_file: .env
```

Comment:
This service is optional. In Spring Boot 3.2 version and above, Docker compose plugin can work even without this part